### DISCUSSION TITLE
When will we see the other apps light up?

### DISCUSSION THREAD

**Discussion by laublarsen** [microsoft/Templates-for-Power-Platform#16] [Q&A] [answered]
There's mentioned 2 more HR Apps. When can we expect these in preview?

thanks

**Comment by EllenWehrle** [comment, marked as answer]
Hello! Right now, the goal is to publish a couple of new HR apps within the next few weeks.

**Comment by laublarsen** [comment]
Excellent. Looking forward to it

/Lau


Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Ryanb58** [reply]
Hey @laublarsen 

Wanted to reach out to let you know that we recently released a new HR App called the Awards and Recognition. Please check it out when you have a minute.

https://github.com/microsoft/Templates-for-Power-Platform/releases

**Reply by laublarsen** [reply]
Hi,

The App is not available for Europe.

I tried to install and nothing happened. I can see in Developer tools that the app is not available.

![image](https://github.com/microsoft/Templates-for-Power-Platform/assets/135946519/74bdcc4c-7adb-40d7-8944-7c779b3569da)

**Reply by mhuguet** [reply]
@laublarsen, do you still need some assistance or shall I close this discussion?
